freud/cluster.pyx

```diff
@@ -229,15 +240,19 @@ cdef class ClusterProperties(_Compute):
     def gyrations(self):
         """(:math:`N_{clusters}`, 3, 3) :class:`numpy.ndarray`: The gyration
         tensors of the clusters."""
+        warnings.warn("This property is deprecated and will be removed",
+                      FreudDeprecationWarning)
         return freud.util.make_managed_numpy_array(
-            &self.thisptr.getClusterGyrations(),
+            &self.thisptr.getClusterInertiaMoments(),
             freud.util.arr_type_t.FLOAT)
 
     @_Compute._computed_property
-    def radii_of_gyration(self):
-        """(:math:`N_{clusters}`,) :class:`numpy.ndarray`: The radius of
-        gyration of each cluster."""
-        return np.sqrt(np.trace(self.gyrations, axis1=-2, axis2=-1))
+    def mom_inertia(self):
```

Comment:
I would prefer to avoid abbreviations. Consider matching coxeter as well: https://coxeter.readthedocs.io/en/latest/package-shapes.html#coxeter.shapes.Ellipsoid.inertia_tensor
```suggestion
    def moments_of_inertia(self):
```
Alternative:
```suggestion
    def inertia_tensors(self):
```